### ISSUE TITLE
Android/ Allow location setting lacks visual separation

### ISSUE THREAD

**Issue by helmines25** [fmidev/weather-app#1013]
The Allow location setting and edit button shift too close together and lack visual separation. The edit button still works as it should, but it may be difficult to distinguish as an interactive element separate from the text. In addition, **part of the "edit" text is cut off in the Swedish and Finnish language versions**. The issue may affect readability and understandability of the setting.

**WCAG 1.4.4, AA**
_Ensure that all text can be resized up to 200% without the loss of content or functionality_

Steps to reproduce:

- Enable larger font and display sizes in the Android device's settings
- Open the weather app
- Navigate to Other --> Settings
- Inspect the Allow location- setting
- Switch the app's language to Swedish or Finnish, inspect how the text behaves as a result

---

Screenshot (app's English language version):

<img width="366" src="https://github.com/user-attachments/assets/d598608d-874f-4eeb-a179-84e28844d231" />

---

Screenshot (app's Swedish language version):

<img width="366" src="https://github.com/user-attachments/assets/13398dfe-11cc-4b23-8225-dab47868ea42" />

---

Screenshot (app's Finnish language version):

<img width="366" src="https://github.com/user-attachments/assets/74729605-b8f5-458f-afe3-a67e74f9e323" />

**Comment by geosaaga** [comment]
This has been fixed in version 6.0.8
